openshift-201/post-outage-checkup.md

```diff
@@ -0,0 +1,182 @@
+# Post Outage Check Up
+
+## Objectives:
+
+After completing this section, you should know what to do after there is an outage and how to handle common problems that may occur.
+
+## Prerequisites
+All resources created in this lab should include your username so you do not clash with other lab participants.  If you set the `$USERNAME` environment variable to your username then you can easily copy/paste the commands below.  Just ensure your username contains only '-', '.' or lowercase alphanumberic characters.
+
+example:
+```bash
+export USERNAME=jmacdonald
+```
+
+## Setup
+We will setup a sample application and proceed with what steps should be taken after an outage occurs.
+
+### Create a new application 
+```bash
+ oc new-app --name crash-app-$USERNAME \
+-i redhat-openjdk18-openshift:1.8 \
+ --context-dir=openshift-201/materials/post-outage-checkup/crash-app \
+ https://github.com/BCDevOps/devops-platform-workshops
+
+```
```

Comment:
```suggestion
```bash
oc new-app --name crash-app-$USERNAME \
-i redhat-openjdk18-openshift:1.8 \
--context-dir=openshift-201/materials/post-outage-checkup/crash-app \
https://github.com/BCDevOps/devops-platform-workshops
```
```

Reply:
yea im removing the empty line and extra spaces in the front of each line. But im not sure about the closing ```, it should still be there. But i'd say it's fine to ignore this change!